build.gradle

```diff
@@ -85,14 +85,6 @@ allprojects {
         module.inheritOutputDirs = true
     }
 
-    repositories {
-        mavenCentral()
-        maven { url 'https://repo.eclipse.org/content/groups/releases' }
-        maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
-        maven { url = "https://s3-eu-west-1.amazonaws.com/maven.seqera.io/releases" }
-        maven { url = "https://s3-eu-west-1.amazonaws.com/maven.seqera.io/snapshots" }
-    }
-
```

Comment:
This should not be removed instead `	maven { url 'https://jitpack.io' }` should be added

Reply:
I had to move all of the repositories config to `settings.gradle` because the `dependencyResolutionManagement` recommended by jitpack was not recognized here

Reply:
It should be enough adding `maven { url 'https://jitpack.io' }` here and nothing else

Reply:
Turns out the includeGroup for jitpack was too strict. Local build is now working with jitpack. I will now see if I can move the config back into the build.gradle